proposals/date_replace.json

```diff
@@ -0,0 +1,99 @@
+{
+    "id": "date_replace",
+    "summary": "Replaces a part of a date",
+    "description": "Replaces a part of a given date string with a new value based on the provided unit, allowing for easy replacing of dates.",
+    "categories": [
+        "date & time"
+    ],
+    "experimental": true,
+    "parameters": [
+        {
+            "name": "date",
+            "description": "The date (and optionally time) to replace.\n\nIf the given date doesn't include the time, the process assumes that the time component is `00:00:00Z` (i.e. midnight, in UTC). The millisecond part of the time is optional and defaults to `0` if not given.",
+            "schema": [
+                {
+                    "type": "string",
+                    "format": "date-time",
+                    "subtype": "date-time"
+                },
+                {
+                    "type": "string",
+                    "format": "date",
+                    "subtype": "date"
+                }
+            ]
+        },
+        {
+            "name": "value",
+            "description": "The period of time in the unit given to replace the existing value.",
+            "schema": {
+                "type": "integer"
+            }
+        },
+        {
+            "name": "unit",
+            "description": "The unit for the value given. The following pre-defined units are available:\n\n- millisecond: Milliseconds\n- second: Seconds .\n- minute: Minutes\n- hour: Hours\n- day: Days - changes only the the day part of a date\n- month: Months\n- year: Years\n\nReplacements with the unit `year`, `month` or `day` do never change the time.",
+            "schema": {
+                "type": "string",
+                "enum": [
+                    "millisecond",
+                    "second",
+                    "minute",
+                    "hour",
+                    "day",
+                    "month",
+                    "year"
+                ]
+            }
+        }
+    ],
+    "returns": {
+        "description": "The date with replaced value. If a time component was given in the parameter `date`, the time component is returned with the date.",
```

Comment:
I think a time component should also be returned when given date was just subtype `date`, but component to replace was e.g. "hour"